frontend/src/plate/functions/lex-specialis/utfall-score.test.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, expect, it } from 'bun:test';
+import { SET_DELIMITER } from '@app/components/smart-editor-texts/types';
+import { EXCLUDE_SCORE, NEUTRAL_SCORE } from '@app/plate/functions/lex-specialis/scores';
+import { MAX_UTFALL_SCORE, getUtfallScore } from '@app/plate/functions/lex-specialis/utfall-score';
+import { UtfallEnum } from '@app/types/kodeverk';
+
+describe('utfall score', () => {
+  it('one case utfall should match same text utfall perfectly', () => {
```

Comment:
Jævla nitpicky, men jeg mener fortsatt at testnavnet skal henge sammen med ordet "it". Eventuelt bare refaktorer til `test`?